src/exports/generated/CBL.gnu

```diff
@@ -1,244 +1,295 @@
-		CBLAuth_NewPassword;
-		CBLAuth_NewSession;
-		CBLAuth_Free;
-		CBLReplicator_New;
-		CBLReplicator_Config;
-		CBLReplicator_Start;
-		CBLReplicator_Stop;
-		CBLReplicator_SetHostReachable;
-		CBLReplicator_SetSuspended;
-		CBLReplicator_Status;
-		CBLReplicator_PendingDocumentIDs;
-		CBLReplicator_IsDocumentPending;
-		CBLReplicator_AddChangeListener;
-		CBLReplicator_AddDocumentReplicationListener;
-		CBLDefaultConflictResolver;
-		kFLNullValue;
-		kFLEmptyArray;
-		kFLEmptyDict;
-		FLSlice_Equal;
-		FLSlice_Compare;
-		FLSlice_Hash;
-		FLSlice_Copy;
-		FLSlice_ToCString;
-		FLSliceResult_New;
-		_FLBuf_Retain;
-		_FLBuf_Release;
-		FLDoc_FromResultData;
-		FLDoc_FromJSON;
-		FLDoc_Release;
-		FLDoc_Retain;
-		FLDoc_GetData;
-		FLDoc_GetRoot;
-		FLDoc_GetSharedKeys;
-		FLData_Dump;
-		FLDump;
-		FLDumpData;
-		FLValue_FromData;
-		FLValue_GetType;
-		FLValue_IsInteger;
-		FLValue_IsUnsigned;
-		FLValue_IsDouble;
-		FLValue_IsEqual;
-		FLValue_AsBool;
-		FLValue_AsData;
-		FLValue_AsInt;
-		FLValue_AsUnsigned;
-		FLValue_AsFloat;
-		FLValue_AsDouble;
-		FLValue_AsString;
-		FLValue_AsArray;
-		FLValue_AsDict;
-		FLValue_AsTimestamp;
-		FLValue_ToString;
-		FLValue_ToJSON;
-		FLValue_ToJSONX;
-		FLValue_ToJSON5;
-		FLValue_FindDoc;
-		FLValue_Retain;
-		FLValue_Release;
-		FLData_ConvertJSON;
-		FLJSON5_ToJSON;
-		FLArray_Count;
-		FLArray_IsEmpty;
-		FLArray_Get;
-		FLArray_AsMutable;
-		FLArray_MutableCopy;
-		FLArrayIterator_Begin;
-		FLArrayIterator_GetCount;
-		FLArrayIterator_GetValue;
-		FLArrayIterator_GetValueAt;
-		FLArrayIterator_Next;
-		FLMutableArray_New;
-		FLMutableArray_GetSource;
-		FLMutableArray_IsChanged;
-		FLMutableArray_Append;
-		FLMutableArray_Set;
-		FLMutableArray_Insert;
-		FLMutableArray_Remove;
-		FLMutableArray_Resize;
-		FLMutableArray_GetMutableArray;
-		FLMutableArray_GetMutableDict;
-		FLDict_Count;
-		FLDict_IsEmpty;
-		FLDict_Get;
-		FLDict_GetWithKey;
-		FLDict_AsMutable;
-		FLDict_MutableCopy;
-		FLDictIterator_Begin;
-		FLDictIterator_GetCount;
-		FLDictIterator_GetKey;
-		FLDictIterator_GetKeyString;
-		FLDictIterator_GetValue;
-		FLDictIterator_Next;
-		FLDictIterator_End;
-		FLDictKey_Init;
-		FLDictKey_GetString;
-		FLMutableDict_New;
-		FLMutableDict_GetSource;
-		FLMutableDict_IsChanged;
-		FLMutableDict_Set;
-		FLMutableDict_Remove;
-		FLMutableDict_RemoveAll;
-		FLMutableDict_GetMutableArray;
-		FLMutableDict_GetMutableDict;
-		FLSlot_SetNull;
-		FLSlot_SetBool;
-		FLSlot_SetInt;
-		FLSlot_SetUInt;
-		FLSlot_SetFloat;
-		FLSlot_SetDouble;
-		FLSlot_SetString;
-		FLSlot_SetData;
-		FLSlot_SetValue;
-		FLKeyPath_New;
-		FLKeyPath_Free;
-		FLKeyPath_Eval;
-		FLKeyPath_EvalOnce;
-		FLDeepIterator_New;
-		FLDeepIterator_Free;
-		FLDeepIterator_GetValue;
-		FLDeepIterator_GetKey;
-		FLDeepIterator_GetIndex;
-		FLDeepIterator_GetDepth;
-		FLDeepIterator_SkipChildren;
-		FLDeepIterator_Next;
-		FLDeepIterator_GetPath;
-		FLDeepIterator_GetPathString;
-		FLDeepIterator_GetJSONPointer;
+		CBL_Retain
;
```

Comment:
Is the ^M carriage return character? It's weird that it shows up before semi-colon. Shall this be cleaned up?

Reply:
Yes!  I wonder when this snuck in....probably from the rebase